test/test_quantization.py

```diff
@@ -695,10 +696,49 @@ def test_single_layer(self):
             [self.calib_data],
             inplace=False)
         result_eager = model_eager(self.calib_data[0][0])
-        torch._C._jit_pass_quant_fusion(model_script._c._get_module('fc1')._get_method('forward').graph)
-        result_script = model_script._c._get_method('forward')(self.calib_data[0][0])
+        result_script = get_forward(model_script._c)(self.calib_data[0][0])
         self.assertEqual(result_eager, result_script)
 
+    @unittest.skip("quantization for inlined linear is not working right now")
+    def test_nested(self):
```

Comment:
It seems that we had more tests before the move, could you please explain why they were changed?

Reply:
the conv+bn test is added later after we fix the conv patterns